tests/letstest/scripts/test_tests.sh

```diff
@@ -10,9 +11,11 @@ $LE_AUTO --os-packages-only
 LE_AUTO_SUDO="" VENV_PATH="$VENV_NAME" $LE_AUTO --no-bootstrap --version
 . $VENV_NAME/bin/activate
 
+PIP_INSTALL=$("$REPO_ROOT/tools/readlink.py" "$REPO_ROOT/tools/pip_install.py")
```

Comment:
Approving because sure it'll work but curious -- why do we need the readlink?

Reply:
It was to get the absolute path to pip_install.py before we moved to a temporary directory but a much saner thing to do is to just install pytest before changing directories.

I made that change, ran test farm tests again, and they still pass.